exercises/palindrome-products/PalindromeProductsExample.swift

```diff
@@ -51,17 +46,17 @@ struct PalindromeProducts {
             }
             results[advanceByIndex] = multiplicationsTemp
```

Comment:
I think we can fix it for sure by creating a "write thread", so in line 47 we dispatch a write queue that we use to write to the results.  

http://stackoverflow.com/questions/24045895/what-is-the-swift-equivalent-to-objective-cs-synchronized